### PR TITLE
Add docs to IMA LockAndData contracts

### PR DESCRIPTION
**Pull request by cstrangedk**
Note: 

- MoneyReceived -> ETHReceived
- SendedERC20 -> SentERC20
- SendERC721 -> SentERC721
- ReceiveERC721 -> ReceivedERC721

### CODE REVIEW

**Review by yavrsky — DISMISSED** [review]

**Review by sergiy-skalelabs — DISMISSED** [review]

File: proxy/contracts/predeployed/LockAndDataForSchainERC20.sol [hunk 1]
```diff
@@ -43,19 +54,37 @@ contract LockAndDataForSchainERC20 is PermissionsForSchain {
         // solium-disable-previous-line no-empty-blocks
     }
 
+    /**
+     * @dev Allows ERC20Module to send (mint) ERC20 tokens to LockAndDataForSchainERC20.
```
Comment (payvint) [inline proxy/contracts/predeployed/LockAndDataForSchainERC20.sol]: to send mint ERC20 tokens **FROM** LockAndDataForSchainERC20

File: proxy/contracts/predeployed/LockAndDataForSchainERC721.sol [hunk 2]
```diff
@@ -44,19 +56,40 @@ contract LockAndDataForSchainERC721 is PermissionsForSchain {
         // solium-disable-previous-line no-empty-blocks
     }
 
+    /**
+     * @dev Allows ERC721Module to send (mint) an ERC721 token to LockAndDataForSchainERC721.
```
Comment (payvint) [inline proxy/contracts/predeployed/LockAndDataForSchainERC721.sol]: FROM

Comment (cstrangedk) [reply proxy/contracts/predeployed/LockAndDataForSchainERC721.sol]: fixed

File: proxy/contracts/predeployed/LockAndDataForSchainERC20.sol (hunk 1 above)
Comment (cstrangedk) [reply proxy/contracts/predeployed/LockAndDataForSchainERC20.sol]: fixed

**Review by payvint — APPROVED** [review]
Good

**Review by sergiy-skalelabs — APPROVED** [review]

**Pull request merged**